src/Api.js

```diff
@@ -30,6 +31,7 @@ class API extends AppDevAPI {
             HelloWorldRouter,
             PlacesAutocompleteRouter,
             RouteRouter,
+            MultiRouteRouter,
```

Comment:
place this after `HelloWorldRouter` for alphabetical order

Reply:
+1